ideascube/forms.py

```diff
@@ -69,11 +69,24 @@ def save(self, *args, **kwargs):
 class UserImportForm(forms.Form):
     source = forms.FileField(required=True)
 
+    def canonicalize_row(self, row):
+        result = {}
+        data_fields = User.get_data_fields()
+        data_fields = {str(f.verbose_name): f.name for f in data_fields}
```

Comment:
I started with the assumption that if we used the `verbose_name` for export, then there must be a good reason, and as such I made the import capable of getting back to the canonical field name.

But let's challenge this assumption, maybe it is the export which should always use the canonical field name?